### ISSUE TITLE
[SUG] Add Custom Images for Channel Points

### ISSUE THREAD

**Issue by KiokuVT** [gezelio/YTCR#67]
Would it be possible to add custom PNG images to change the look of each Redeem, and possibly the Points button itself? So if i want to rename and add an image for my channel points to similar to Twitches?
![image](https://github.com/gezelio/YTCR/assets/135864218/d4580e96-9486-4acb-946e-261ab50330e3)
![image](https://github.com/gezelio/YTCR/assets/135864218/657b2356-c078-4563-ab09-e9cbca0a0775)

**Comment by tommerty** [comment]
Yeah so this is something we've heavily considered but the problem comes with moderating uploaded images and such! This might be something down the line, but I think there's a lot more things that could be added first. I'll place this on a backburner
